fineract-provider/src/main/java/org/apache/fineract/notification/domain/Notification.java

```diff
@@ -90,7 +90,7 @@ public Long getActor() {
     }
 
     public void setActor(Long actor) {
-        this.actorId = actorId;
+        this.actorId = actor;
     }
```

Comment:
@vorburger concerning org.apache.fineract.notification.domain.Notification.setActor(Long) error prone picked this up locally

Reply:
@percyashu nice! FYI I've also fixed this in #858 ...